dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
1. There is `\W` for `not \w`
2. `+` inside `[]` is just a plus sign
3. Isn't `\)` is part of _not_ any word characters? Also may be for clarity it's better to use another regex in the list (would be slower), that handles matching "(...)" instead of "(..." or "...)".